diff --git a/proton-c/CMakeLists.txt b/proton-c/CMakeLists.txt
index 0fec525..d4dbc67 100644
--- a/proton-c/CMakeLists.txt
+++ b/proton-c/CMakeLists.txt
@@ -415,7 +415,7 @@ add_library (
   ${qpid-proton-extra-deps}
   )

-target_link_libraries (qpid-proton ${UUID_LIB} ${SSL_LIB} ${SASL_LIB} ${TIME_LIB} ${PLATFORM_LIBS})
+target_link_libraries (qpid-proton uuid ${SSL_LIB} ${SASL_LIB} rt ${PLATFORM_LIBS})

 set_target_properties (
   qpid-proton
